addons/microsoft_calendar/models/microsoft_sync.py

```diff
@@ -177,13 +172,6 @@ def _write_from_microsoft(self, microsoft_event, vals):
     def _create_from_microsoft(self, microsoft_event, vals_list):
         return self.with_context(dont_notify=True).create(vals_list)
 
-    @api.model
-    @ormcache_context('uids', keys=('active_test',))
-    def _from_uids(self, uids):
```

Comment:
Do not remove functions in stable version.

Reply:
Done, reverted change :D